articles/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -98,28 +98,43 @@ The following is a sample guide for use with CloudFlare **only**; please note, t
 ## Step by Step Guide with AWS CloudFront
 
```

Comment:
At the top of the file doc, we talk of CDN providers and mention Cloudflare or Fastly. Should we also include CloudFront there. Lot of our customers are in AWS and might be using CloudFront

Reply:
Great catch - adding.